src/Sulu/Bundle/SearchBundle/Controller/WebsiteSearchController.php

```diff
@@ -73,7 +73,7 @@ public function __construct(
      */
     public function queryAction(Request $request)
     {
-        $query = $this->getRequestParameter($request, 'q', true);
+        $query = $this->getRequestParameter($request, 'q', true, '');
```

Comment:
force should be false in this case:

```suggestion
        $query = $this->getRequestParameter($request, 'q', false, '');
```